yarn-project/end-to-end/src/e2e_token_contract/reading_constants.test.ts

```diff
@@ -15,28 +17,13 @@ const toString = (val: bigint[]) => {
 describe('e2e_token_contract reading constants', () => {
   const t = new TokenContractTest('reading_constants');
   const { TOKEN_DECIMALS, TOKEN_NAME, TOKEN_SYMBOL } = TokenContractTest;
-  // Do not destructure anything mutable.
-  const { logger } = t;
-  let reader: ReaderContract;
+  let { wallets } = t;
 
   beforeAll(async () => {
     await t.applyBaseSnapshots();
-
-    await t.snapshot(
```

Comment:
![Winnie The Pooh Happy Dance GIF by Leon Denise](https://media3.giphy.com/media/l3vRlT2k2L35Cnn5C/giphy.gif)